### ISSUE TITLE
Linting issues 

### ISSUE THREAD

**Issue by xvilo** [sserbin/twig-linter#2]
We tried running your twig linter over our twig 2 codebase, there where some peculiar things and would like to discuss why this happens and the possibility to change this. 

The first thing is that these notations are not supported:
- `same as(SOMEVAR)` -> should be `sameas(SOMEVAR)`. We would like to keep this with space. 
- `divisible by(2)` -> should be `divisibleby(2)`. Again, we would like to keep this with a space.

Also we have some custom funcitons in our environment, it is okay that these wont be linted, but after adjusting the codebase to the above "issues" we still have 2 errors left:

```

  ERROR  in {SOMEFILE}.twig (line 49)
      47                                         {HTML HERE}
      48                                              {HTML HERE}
  >>  49                                                 {{ getCurrencySymbol() }}
  >> Value for argument "_" is required for function "getCurrencySymbol". 
      50                                                 {{  {HTML CUSTOM FUNCTIONS WITH ARGS HERE} }}
      51                                                 {HTML HERE}
```

Not sure why the linter gets confused on this

**Comment by sserbin** [comment]
Thanks for reporting this, @xvilo ! This should be fixed with 4f76e689b321fd16d584635e657fd778ea4a5dee
I'll make a release shortly as well

**Comment by xvilo** [comment]
Love the quick response **and** fix. Please keep up to good work!
